testsuite/features/build_validation/init_clients/proxy.feature

```diff
@@ -52,7 +52,6 @@ Feature: Setup SUSE Manager proxy
     And file "/etc/sysconfig/apache2" should contain "rewrite" on "proxy"
     And file "/etc/sysconfig/apache2" should contain "version" on "proxy"
     And file "/etc/sysconfig/apache2" should contain "ssl" on "proxy"
-    And file "/etc/sysconfig/apache2" should contain "access_compat" on "proxy"
```

Comment:
Did you discuss with dev team the fact that we don't have access_compat in SUMA?
This difference is expected from a design point of view?

Reply:
Not yet. I will ask in our team channel and report back.

Reply:
We will need to wait for feedback from mc once he is back from vacation.

Reply:
Ok, mc confirmed that we do not use `access_compat` anymore. This was something for Apache 2.2 which we do not support since a long time anymore. So it is fine to remove the line from the test.